docs/articles/systemd.md

```diff
@@ -98,7 +98,7 @@ directory:
 
     [Service]
     ExecStart=
-    ExecStart=/usr/bin/docker daemon -H fd:// --graph /mnt/docker-data --storage-driver btrfs
+    ExecStart=/usr/bin/docker daemon -H fd:// --graph /mnt/docker-data --storage-driver=overlay
```

Comment:
You can `git commit --amend’, and force push, that should update the pull request, and not add a new commit